swig/fnvector-getarraypointer.patch

```diff
@@ -1,8 +1,1721 @@
+diff --git a/src/nvector/manyvector/fmod/fnvector_manyvector_mod.f90 b/src/nvector/manyvector/fmod/fnvector_manyvector_mod.f90
```

Comment:
This patch is extra large, primarily due to whitespace differences.  I suppose that users won't actually *look* at this file, but if it could instead be generated while ignoring whitespace, then it would be more focused on the actual changes.

Reply:
Huh, it wasnt originally this large. I will regenerate it.

Reply:
Regenerated w/o whitespace and it is much smaller now.